package.json

```diff
@@ -86,7 +86,7 @@
     "react-router-dom": "^4.0.0",
     "react-test-renderer": "^16.5.2",
     "stylelint": "^9.0.0",
```

Comment:
We'll want to update this to `^10.0.1` to match the peerDependency needed for stylelint-config-terra v3
https://github.com/cerner/stylelint-config-terra/blob/master/package.json#L53

Reply:
Resolved in this commit: https://github.com/cerner/terra-core/pull/2379/commits/e716325a4d0665700769eff3629a03e9d8f5a7cd